dvc/commands/add.py

```diff
@@ -54,11 +23,7 @@ def run(self):
                 fname=self.args.file,
                 external=self.args.external,
                 glob=self.args.glob,
-                desc=self.args.desc,
```

Comment:
@dberenbaum, do we also want to get rid of `desc`?

Reply:
Let's drop it from `dvc add`. We could add support for all these flags and make them write artifact info to `dvc.yaml` in that case, but that's probably a separate issue.

I assume `.dvc` files with descriptions won't break for regular data operations, right? In other words, we still allow for that field even if we don't use it at all?

Reply:
All of the changes are for CLI. This PR affects add/import/import-url. It does not break .dvc or dvc.yaml files.